airflow/configuration.py

```diff
@@ -322,6 +339,29 @@ def __init__(self, defaults, *args, **kwargs):
         ConfigParser.__init__(self, *args, **kwargs)
 
     def get(self, section, key, **kwargs):
+        return self._get(section, key, True)
```

Comment:
what are kwargs and why are you not passing them through?  (I don't know python, sorry)